lib/appoptics_apm/inst/graphql.rb

```diff
@@ -78,41 +78,40 @@ def gql_config
             ::AppOpticsAPM::Config[:graphql] ||= {}
           end
 
-          ###
-          # any errors graphql has dealt with are added to the response
-          def report_errors(result)
-            return unless result.is_a?(Array)
+          def transaction_name(query)
+            return if gql_config[:transaction_name] == false ||
+                      ::AppOpticsAPM::SDK.get_transaction_name
 
-            result.each do |res|
-              if res.is_a?(GraphQL::Query::Result) && res.to_h['errors']
-                msg = res.to_h['errors'].map { |r| r['message'] }.join("\n")
-                AppOpticsAPM::SDK.log_info(Message: "GraphQL Errors:\n#{msg}")
-              end
-            end
+            split_query = query.strip.split(/\W+/, 3)
+            split_query[0] = 'query' if split_query[0].empty?
+            name = "graphql.#{split_query[0..1].join('.')}"
+
+            ::AppOpticsAPM::SDK.set_transaction_name(name)
           end
 
-          def maybe_set_transaction_name(query)
+          def multiplex_transaction_name(names)
             return if gql_config[:transaction_name] == false ||
                       ::AppOpticsAPM::SDK.get_transaction_name
 
-            split_query = query.strip.split(/\W+/, 3)
-            name = "graphql.#{split_query[0..1].join('.')}"
+            name = "graphql.multiplex.#{names.join('.')}"
+            name = "#{name[0..251]}..." if name.length > 254
```

Comment:
transaction name length would be 256 including the # and ..., should it be [0..250]?

Reply:
`#` is part of the expression operator `#{}`, 
e.g. `"three plus three is #{3+3}"` 
-> "three plus three is 3"

Reply:
okay, thanks for confirming, I didn't realize the # was part of the expression